projects/plugins/social/src/js/components/header/index.js

```diff
@@ -31,6 +32,7 @@ const Header = () => {
 			sharesCount: select( STORE_ID ).getSharesCount(),
 			postsCount: select( STORE_ID ).getPostsCount(),
 			isShareLimitEnabled: select( STORE_ID ).isShareLimitEnabled(),
+			hasPaidPlan: select( STORE_ID ).hasPaidPlan(),
```

Comment:
Super minor thing, but should these be `store.` instead of `select( STORE_ID ).`? We already have the store defined.